visualizer/src/service/apiMachine.js

```diff
@@ -85,10 +108,27 @@ const createApiMachine = ({ projectId, bucket }) =>
             },
           },
         },
```

Comment:
Another machine viz. The change is the new downloading state

<img width="536" alt="Screen Shot 2021-10-20 at 10 34 57 AM" src="https://user-images.githubusercontent.com/40068392/138143037-f1edcc8a-03ca-49a5-b549-f5be26fd057a.png">